packages/discovery-react-components/src/components/SearchFacets/components/FieldFacets.tsx

```diff
@@ -117,7 +117,9 @@ export const FieldFacets: FC<FieldFacetsProps> = ({
         }
 
         const hasCategories =
-          aggregation.field.includes('enriched_') && aggregation.field.includes('entities.text');
+          aggregation.field.includes('enriched_') &&
+          aggregation.field.includes('entities.text') &&
+          aggregation.results![0].aggregations !== undefined;
```

Comment:
Can we be positive that `results` exists? Seems like it would be better to do optional chaining here if we can. 

```suggestion
          aggregation.results?.[0]?.aggregations !== undefined;
```

Reply:
Even if we cannot, I dont think i am comfortable with using the `!`. We have seen too many times that the API can return unexpected results.

Reply:
Agreed. I used `get` instead since optional chaining is not available on our current version of `TypeScript`. This is ready for a re-review.

Reply:
It isn't? I thought I fixed this a while back: https://github.com/watson-developer-cloud/discovery-components/pull/101

Reply:
!!! Looking at that, and it looks like that PR never got merged into `master`. It went into `develop` and I'm not seeing those changes reflected in `master`.

Reply:
Huh, did I accidentally set the base to `develop`? Well, color me embarrassed! I'll put up another PR.

Reply:
Looks like we switched from `develop` to `master` on Mar 26, same day that PR went in. Must have gotten lost in the shuffle.

Reply:
I was thinking it was probably something like that.

Reply:
=> https://github.com/watson-developer-cloud/discovery-components/pull/139